foxtrot-core/src/main/java/com/flipkart/foxtrot/core/querystore/impl/ElasticsearchUtils.java

```diff
@@ -63,6 +73,35 @@ public static String getIndexPrefix(final String table) {
                 ElasticsearchUtils.TABLENAME_PREFIX, table, ElasticsearchUtils.TABLENAME_POSTFIX)};
     }
 
+    public static String[] getIndices(final String table, final ActionRequest request) throws Exception {
+        final Interval interval = new PeriodSelector(request.getFilters()).analyze();
+        DateTime start = interval.getStart().toLocalDate().toDateTimeAtStartOfDay();
+        if(start.getYear() == 1970) {
+            logger.warn("Request of type {} running on all indices", request.getClass().getSimpleName());
+            return getIndices(table);
+        }
+        DateTime end = interval.getEnd().plusDays(1).toLocalDate().toDateTimeAtStartOfDay();
+        List<String> indices = Lists.newArrayList();
+        String lastIndex = "";
+        String index = "";
+        while (true) {
+            index = String.format("%s-%s-%s-%s",
+                    ElasticsearchUtils.TABLENAME_PREFIX, table, ElasticsearchUtils.TABLENAME_POSTFIX,
```

Comment:
It's not the same. I'm not using a timestamp here. IT's directly coming from the DateTime object